src/index.html

```diff
@@ -15,8 +15,90 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="./images/logo.png"
+          class="logo"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              href="#"
+              class="nav__link apple__link is-active"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class name 'apple__link' doesn't seem to represent the content. It would be better to use a more descriptive class name.